planner/core/src/main/java/eu/seaclouds/platform/planner/core/HttpHelper.java

```
@@ -83,22 +84,48 @@ public String getRequest(String restPath, List<NameValuePair> params){
         }
     }
 
+    public String postInBody(String restPath, String bodyContent){
+        HttpPost post = new HttpPost(prepareRequestURL(restPath));
+        try {
+            StringEntity entity = new StringEntity(bodyContent);
+            post.setEntity(entity);
+            HttpResponse response = httpclient.execute(post);
+            String result = EntityUtils.toString(response.getEntity());
+            return result;
+        }catch (ClientProtocolException e) {
+            return null;
+        } catch (UnsupportedEncodingException e) {
+            return null;
+        } catch (IOException e) {
+            return null;
```

Don't ignore exceptions